frogbot/frogbot.py

```diff
@@ -0,0 +1,40 @@
+import os,sys
+import yaml
+
+def create_frogbot_config_file(change_modules_path,repo_name,source_branch_name):
+
+    data = [{
+        'params': {
+            'git': {
+                'branches': [source_branch_name],
+                'repoName': repo_name
+            },
+            'scan': {
+                'projects': [
+                    {
+                        'workingDirs': change_modules_path
+                    }
+                ]
+            }
+        }
+    }]
+    folder_name = '.frogbot'
```

Comment:
This script should take the workspace path to make sure the config file get created in its root. For whatever reason, if we go into a subfolder, the file will get created in there.

Reply:
Yes we tested

Reply:
If someone changes the composite to put a `cd subfolder` before calling the python script, what would happen? Where would the file be created?

Reply:
For these we have changed the code